### ISSUE TITLE
onLongPress is not a required function

### ISSUE THREAD

**Issue by xiaomsh** [salah-rashad/flutter_context_menu#6]
There has a simple requirement
right click to show menu and long press to multiselect

Can you add a field    
supportLongPress   or something else 
to enable the onLongPress function in GestureDetector

**Comment by salah-rashad** [comment]
It seems I might have missed some details. To clarify, are you suggesting making the 'longTapPressed' optional?

Could you share additional details, code snippets, or even screenshots of the context you're working in? This will help me better understand the situation.

Thank you

**Comment by wenchaosong** [comment]
@salah-rashad  yes  I mean make the longTapPressed optional , because sometimes need long press to do other things
